docs/faq.md

```diff
@@ -2,6 +2,7 @@
 
 ## How can I contact the developers?
 
+- [Discord channel](https://discord.gg/qSRxH4Xu).
 - [Community forum](https://www.percona.com/forums/questions-discussions/percona-monitoring-and-management).
```

Comment:
Maybe reverse discord with forum?  would prefer people go to forums where expectation of help is a little more asynchronous while discord is real-time expectation.

Reply:
Will do after merge